velox/dwio/common/Options.h

```diff
@@ -389,6 +389,7 @@ class ReaderOptions : public io::ReaderOptions {
   bool fileColumnNamesReadAsLowerCase{false};
   bool useColumnNamesForColumnMapping_{false};
   std::shared_ptr<folly::Executor> ioExecutor_;
+  bool enableParquetBloomFilter_{false};
```

Comment:
I saw @yingsu00 had a comment about this; can we have the bloom filter stuff being a Parquet option, and not a general dwio option (since this is specific to Parquet)?

Reply:
I renamed this to generic `readBloomFilter_`, so it can be used for any format. Also, I'm currently initializing this only for parquet in `configureReaderOptions`.